app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextModuleActions.java

```diff
@@ -381,4 +413,58 @@ public void onClick(DialogInterface dialog, int id) {
 
         builder.show();
     }
+
+    /**
+     * Show the camera picker via intent
+     * Source: http://developer.android.com/training/camera/photobasics.html
+     */
+    private void showCameraDialog() {
```

Comment:
Put this method into `opoc/ShareUtil`. Put anything missing as a method parameter, I will do the refactor of the method later myself :). Use then: ```_cameraPictureFilepath = ShareUtil.requestPicture(params)```